stablehlo/reference/Interpreter.cpp

```diff
@@ -58,6 +58,18 @@ llvm::Expected<SmallVector<Tensor>> eval(func::FuncOp func,
       if (it != stackFrame.end()) return it->second;
       llvm::report_fatal_error("Expected a terminator when evaluating func");
     };
+    auto fetchVariadic = [&](OperandRange range) -> ArrayRef<Tensor> {
+      SmallVector<Tensor> arr;
+      for (auto value : range) {
+        auto it = stackFrame.find(value);
+        if (it != stackFrame.end()) {
```

Comment:
can we remove braces from both if else statements

Reply:
maybe move the statements inside the block next to if else if there is enough space?

Reply:
Updated the changes for your first comment. The second comment will not be allowed by the formatter we are using.